### ISSUE TITLE
Is aria-flowto an option?

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y-extended-desc#18]
(Though even if so, we are not sure if the support is good, as with aria-details.)

**Comment by gautierchomel** [comment]
This is missing and has to be added to the document "Considered alternatives" section. There is indeed a lack of support for `aria-flowto` but the main issue is due to  aria-flowto being a generic reading-order tool.  Using `aria-flowto` to jump to an extended description would disrupt the natural reading order, forcing users off course. The document specifically wants to avoid "breaking reading flow.

**Comment by gautierchomel** [comment]
Proposed additional text:

- `aria-flowto`: designed to suggest an alternate linear reading sequence (redirecting standard navigation flow), whereas extended descriptions require a programmatic association to supplementary metadata that preserves the primary reading context.

**Comment by gautierchomel** [comment]
Suggest to change "supplementary metadata" to  "supplementary content".
